libs/tracing/unix/Tracing.ml

```diff
@@ -51,6 +51,13 @@ module Otel = Opentelemetry
  * TODO we'll probably need instructions for some system of tags?
  *)
 
+(*****************************************************************************)
+(* Constants *)
+(*****************************************************************************)
+
+let default_endpoint = "https://telemetry.dev2.semgrep.dev"
```

Comment:
```suggestion
let default_endpoint = "https://telemetry.semgrep.dev"
```

Reply:
Suggest making prod the default and requiring explicit, additional configuration to send to non-prod.

Reply:
I'm going to leave it as dev for now since it's still in progress but I'll make a ticket to switch it